### ISSUE TITLE
Converting 1 raw to NANO gives 0

### ISSUE THREAD

**Issue by Joohansson** [marvinroger/nanocurrency-js#74]
It seems it does not support more decimals than 20. The result is 0 until you reach 10000000000 raw then it prints out 0.00000000000000000001. Same thing with https://github.com/numsu/nanocurrency-web-js.

Is that some kind of FLOAT limitation? I would like 1 raw to print all 30 zeroes (or more).

**Comment by marvinroger** [comment]
Can you change this line:

https://github.com/marvinroger/nanocurrency-js/blob/1eea968879bb90aae44a6db28914f9f3f74609c0/packages/nanocurrency/src/conversion.ts#L10

To:

```js
const TunedBigNumber = BigNumber.clone({ EXPONENTIAL_AT: 1e9, DECIMAL_PLACES: 1e9 })
```

And report if it worked?

**Comment by Joohansson** [comment]
I installed this library in a reactJS app with "npm install" and I only have the file nanocurrency.cjs.js, which I did the change as suggested above. But I'm sure if the app will use the new code? I'm still seeing the same. I'm not super familiar with how npm works.

**Comment by Joohansson** [comment]
Your change is confirmed working. I ended up importing conversion.ts and check.ts directly into the react app (instead of using the `import * as nano from 'nanocurrency'` via NPM and nano.convert) and made the modifications needed. Now using `import * as convert from './conversion'` and convert.convert

**Comment by besoeasy** [comment]
works well no issuue, can be tuned for personal use too !

issue can be closed !

**Comment by marvinroger** [comment]
I'll fix and release a fix very soon, hopefully tomorrow! Thanks for the feedback. 😉

**Comment by marvinroger** [comment]
Fixed in 2.4.0, thanks. 😉
